---
float64[] angles
std_msgs/Bool success